src/org/zaproxy/zap/extension/websocket/manualsend/ManualWebSocketSendEditorDialog.java

```diff
@@ -187,32 +217,58 @@ public void unload() {
 		getRequestPanel().unload();
 	}
 	
-	private static final class WebSocketMessagePanel extends JPanel {
+	private static final class WebSocketMessagePanel extends JPanel{
 
 		private static final long serialVersionUID = -3335708932021769432L;
 		
+		private static final Logger LOGGER = Logger.getLogger(ManualWebSocketSendEditorDialog.class);
+		
 		private final HttpPanel messagePanel;
 
-		private WebSocketUiHelper wsUiHelper;
-	
+		private JButton btnReopen;
+		private JButton btnReopenEdit;
+		private JPanel reopenPanel;
+		private JLabel connectingLabel;
+		
+        private WebSocketUiHelper wsUiHelper;
+        private WebSocketPanelSender sender;
+        
+        @Deprecated
 		public WebSocketMessagePanel(ChannelSortedListModel channelsModel, JToolBar controlToolbar, HttpPanel messagePanel) throws IllegalArgumentException {
```

Comment:
We can safely remove the constructor (instead of deprecating), the class is private.